### PR TITLE
Initial support for breadcrumbs

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

File: lib/Raven/Breadcrumbs.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    public function is_empty()
+    {
+        return $this->count === 0;
+    }
+
+    public function to_json()
+    {
+        return array(
+            'values' => $this->fetch,
```
Comment (56quarters) [inline lib/Raven/Breadcrumbs.php]: Is this supposed to be the bare `$this->fetch` or a method call `$this->fetch()`?

**Comment by dcramer** [comment]
Need to:

- Write tests for Monolog
- Ensure Monolog support isn't going to affect autoloader and older versions of PHP

File: lib/Raven/Breadcrumbs.php (hunk 1 above)
Comment (dcramer) [inline lib/Raven/Breadcrumbs.php]: good catch :)

(fyi this is def a wip yet)

**Comment by dcramer** [comment]
Alright I think this is generally ok. Will want to squash merge since some of it was broken. I'd def prefer if we could easily do a global handler in Monolog, but the ecosystem is sane enough that it wont be *that* bad to get people to configure it.

**Pull request merged**